src/parser/jobs/gnb/modules/Ammo.tsx

```diff
@@ -144,10 +208,35 @@ export default class Ammo extends Module {
 				},
 			],
 		}
+
+		const chartOptions = {
+			scales : {
+				yAxes: [{
+					ticks: {
+						beginAtZero: true,
+						min: 0,
+						max: 2,
+						callback: ((value: number) => {
```

Comment:
would be good to revisit this and see if you can get it working with the built in step handling